operator/roles/kiali-deploy/tasks/openshift/os-main.yml

```diff
@@ -54,3 +54,8 @@
   include: openshift/os-oauth.yml
   when:
   - is_openshift == True
+
+- name: Process Kiali Console Links on OpenShift
+  include: openshift/os-consolelinks.yml
+  when:
+  - is_openshift == True
```

Comment:
since we are refactoring os-consolelinks.yaml and removing the bulk of it into a common yaml to create kiali_route_url, that means only about 5 lines are left. So we can delete the os-consolelinks.yaml entirely and put those 5 lines here (so rather than doing an `include` just do the `k8s.definition` here.

```
- name: Process OpenShift Console Links
  k8s:
    definition: "{{ lookup('template', 'templates/openshift/console-links.yaml') }}"
  when:
  - is_openshift == True
```